packages/airnode-node/src/evm/gas-prices.ts

```diff
@@ -11,43 +11,79 @@ export interface FetchOptions {
 
 export const parsePriorityFee = ({ value, unit }: PriorityFee) => ethers.utils.parseUnits(value, unit ? unit : 'wei');
 
-export const getGasPrice = async (options: FetchOptions): Promise<LogsData<GasTarget | null>> => {
+const getLegacyGasPrice = async (options: FetchOptions): Promise<LogsData<GasTarget | null>> => {
+  const { provider } = options;
+  const logs = Array<PendingLog>();
+
+  const operation = (): Promise<BigNumber> =>
```

Comment:
@Siegrift This is my take on logging the underlying errors without modifying `go`.

Reply:
I dislike this. We should either modify the `retryOperation` utility to log out unsuccessful attempts or not implement it at all.

I think not implementing it now is preferable.

Reply:
All good, I'll revert.